backend/accounts/baseaccount.go

```diff
@@ -253,6 +255,7 @@ func (account *BaseAccount) ExportCSV(w io.Writer, transactions []*TransactionDa
 		return errp.WithStack(err)
 	}
 
+	isERC20 := account.Coin().IsERC20()
```

Comment:
I think instead if adding a interface function for all coins, the info is already in `transaction.IsERC20`.

Alternatively you could match the specific coin: `ethCoin, ok := account.Coin().(*eth.Coin); isERC20 := ok && ethCoin.ERC20Token() != nil`

Reply:
Yeah this is just a draft pr and it's actually broken right now (doesn't build due to import cycle), so it's going to change

Reply:
> Alternatively you could match the specific coin: `ethCoin, ok := account.Coin().(*eth.Coin); isERC20 := ok && ethCoin.ERC20Token() != nil`

Fun fact, before I noticed that we have `IsERC20` in the tx data, I was doing this, but it's not possible as it creates an import cycle (we can't import `eth` here as `eth` itself imports `accounts`) that's the reason for that workaround of the interface :)